contracts/bond/DaoBondCollateralWhitelist.sol

```diff
@@ -15,6 +15,9 @@ abstract contract DaoBondCollateralWhitelist is Initializable {
         mapping(address => string) symbols;
     }
 
+    event CollateralAdded(address collateralTokens, string symbol);
```

Comment:
nit; keep the event naming the present tense.

Elsewhere you should see the present tense, broadly following OZ's approach to event naming e.g. `Transfer` instead of `Transferred` https://github.com/OpenZeppelin/openzeppelin-contracts/blob/9bded169e8f765f2e9279dca599c2175ce81dbbd/contracts/token/ERC20/IERC20.sol#L75